setup.py

```diff
@@ -31,9 +43,10 @@
     'django-fake-database-backends',
 ]
 
+# noinspection PyUnresolvedReferences
 setup(
     name='django-migration-linter',
-    version='0.0.7',
+    version=get_version(),
```

Comment:
Why can't we just do a `from django_migration_linter.contants import version` and use the `version` here?
All the regex logic above is quite complicated 🤔